### ISSUE TITLE
coingecko version?

### ISSUE THREAD

**Issue by pir8radio** [lewisxhe/CryptocurrencyLib#4]
i see you made a version for 360 api..   check out coingecko API,  they allow like 50 requests a min or something free plan, no api keys.  I use it on a few of my projects.    just working on building an ESP32 ticker, went to use your wrapper and having some issues,  not you im still learning arduino stuff..  but just throwing coingecko out there i would love if someone made a arduino wrapper for them..

**Comment by lewisxhe** [comment]
You mean to make a lib for Coingecko?

**Comment by pir8radio** [comment]
Yes.   or are you about to tell me there is one out there already?  :-)    but yea their api is pretty similar to CMC.
